src/main/java/com/uw/util/UserNameGenerator.java

```diff
@@ -1,26 +1,26 @@
 package com.uw.util;
 
 import com.uw.model.User;
-import com.uw.service.UserService;
+import com.uw.repository.UserRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class UserNameGenerator {
 
-    private UserService userServiceImpl;
+    private UserRepository userRepository;
 
     @Autowired
-    public void setUserServiceImpl(UserService userServiceImpl) {
-        this.userServiceImpl = userServiceImpl;
+    public void setUserServiceImpl(UserRepository userRepository) {
+        this.userRepository = userRepository;
     }
 
     public String generateUsername( String firstName, String lastName ) {
         String username = firstName + "." + lastName;
         int cont = 0;
-        if(!userServiceImpl.findAll().isEmpty()){
-            System.out.println(userServiceImpl.findAll());
-            for( User value: userServiceImpl.findAll()){
+        if(userRepository.findAll().isPresent()){
```

Comment:
A more elegant way could be using reduce: 

`int cont = userRepository.findAll().orElse(EmptyList).stream.reduce(0, user -> value.getFirstName().equalsIgnoreCase(firstName)
                        && value.getLastName().equalsIgnoreCase(lastName))` 

See https://www.baeldung.com/java-stream-reduce